src/Definitions/Security/JWTSecurityDefinitionsGenerator.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Mtrajano\LaravelSwagger\Definitions\Security;
+
+use Mtrajano\LaravelSwagger\DataObjects\Route;
+use Mtrajano\LaravelSwagger\Definitions\Security\Contracts\SecurityDefinitionsGenerator;
+
+class JWTSecurityDefinitionsGenerator implements SecurityDefinitionsGenerator
```

Comment:
This class just implements the definitions to authentication via `jwt` on version `2.0` of the Swagger. 
Exists a config to define the `security_definition_type`. It will be used if the value is `jwt`.